config/initializers/rolify.rb

```diff
@@ -0,0 +1,7 @@
+Rolify.configure do |config|
```

Comment:
Unused block argument - `config`. You can omit the argument if you don't care about it.